deploy/operator.yaml

```diff
@@ -41,10 +41,12 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
-        - name: WATCH_NAMESPACE
+        - name: OPERATOR_NAMESPACE
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: WATCH_NAMESPACE
+          value: ""
```

Comment:
This is for the cluster-scoped changes. Need to update RBAC to use cluster roles and cluster rolebindings.